deploy/006_deploy_offchain_retrieval.ts

```diff
@@ -0,0 +1,31 @@
+import { namehash } from "ethers";
+import { type DeployFunction } from "hardhat-deploy/types";
+import { type HardhatRuntimeEnvironment } from "hardhat/types";
+
+// Deploys a copy of all the contracts
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, ethers } = hre;
+  const { deploy } = deployments;
+  const { deployer, ensRegistry, ensNameWrapper } = await getNamedAccounts();
+
+  const baseDeployArgs = {
+    from: deployer,
+    log: true,
+    autoMine: hre.network.tags.test,
+    deterministicDeployment: !hre.network.tags.test,
+  };
+
+  const ens = await ethers.getContractAt("ENS", ensRegistry);
+  const ensDomainOwner = await ens.owner(namehash("standard-crypto.eth"));
```

Comment:
@gsgalloway will we deploy this under our domain / where should the reverse record resolve to?

Reply:
If we do give the digis wildcard resolver an ENS name, then it'd probably be under our standard-crypto.eth name, yeah.

We get to decide the actual reverse record later, right?

Reply:
That's right